### PR TITLE
Reformat the rest of the adapter charts

### LINKED ISSUE #1912: [Charts] Consistent hyphenation for helm chart names
@Aisuko thanks for this. It would be great if we can revisit the other charts and update their names to include consistent hyphenation (e.g. meshery-consul)

_Originally posted by @leecalcote in https://github.com/layer5io/meshery/pull/1900#discussion_r521516040_

### PR DESCRIPTION
**Pull request by Aisuko**
Signed-off-by: aisuko <urakiny@gmail.com>

**Description**

This PR fixes #1912

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Aisuko** [comment]
```
NAME                                  READY   STATUS    RESTARTS   AGE
pod/meshery-78659f4d65-qfn5q          1/1     Running   0          7m57s
pod/meshery-consul-5578fb5d54-tlgb9   1/1     Running   0          7m3s
pod/meshery-istio-5f9cf5f678-mchnc    1/1     Running   0          7m3s
pod/meshery-kuma-65bc58d67c-t7l52     1/1     Running   0          7m57s
pod/meshery-linkerd-db46fbc88-ccnpb   1/1     Running   0          7m57s
pod/meshery-nsm-d4dbf8f9b-hqc84       1/1     Running   0          7m57s
pod/meshery-osm-654f884b76-ffx84      1/1     Running   0          7m57s

NAME                      TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)     AGE
service/meshery           ClusterIP   10.97.119.203    <none>        8080/TCP    7m57s
service/meshery-consul    ClusterIP   10.97.141.111    <none>        10002/TCP   7m3s
service/meshery-istio     ClusterIP   10.106.102.178   <none>        10000/TCP   7m3s
service/meshery-kuma      ClusterIP   10.110.169.103   <none>        10007/TCP   7m57s
service/meshery-linkerd   ClusterIP   10.104.36.196    <none>        10001/TCP   7m57s
service/meshery-nsm       ClusterIP   10.97.129.64     <none>        10004/TCP   7m57s
service/meshery-osm       ClusterIP   10.103.243.140   <none>        10009/TCP   7m57s

NAME                              READY   UP-TO-DATE   AVAILABLE   AGE
deployment.apps/meshery           1/1     1            1           7m57s
deployment.apps/meshery-consul    1/1     1            1           7m3s
deployment.apps/meshery-istio     1/1     1            1           7m3s
deployment.apps/meshery-kuma      1/1     1            1           7m57s
deployment.apps/meshery-linkerd   1/1     1            1           7m57s
deployment.apps/meshery-nsm       1/1     1            1           7m57s
deployment.apps/meshery-osm       1/1     1            1           7m57s

NAME                                        DESIRED   CURRENT   READY   AGE
replicaset.apps/meshery-78659f4d65          1         1         1       7m57s
replicaset.apps/meshery-consul-5578fb5d54   1         1         1       7m3s
replicaset.apps/meshery-istio-5f9cf5f678    1         1         1       7m3s
replicaset.apps/meshery-kuma-65bc58d67c     1         1         1       7m57s
replicaset.apps/meshery-linkerd-db46fbc88   1         1         1       7m57s
replicaset.apps/meshery-nsm-d4dbf8f9b       1         1         1       7m57s
replicaset.apps/meshery-osm-654f884b76      1         1         1       7m57s
```

File: install/kubernetes/helm/meshery/charts/meshery-istio/Chart.yaml [hunk 1]
```diff
@@ -18,12 +18,12 @@ type: application
...
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.0.0
+version: 2.0.0
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application.
 # TODO: Need the confirm the image tag.
-appVersion:
+appVersion: stable-latest
```
Comment (leecalcote) [inline install/kubernetes/helm/meshery/charts/meshery-istio/Chart.yaml:26]: Nice catch. 👍

File: install/kubernetes/helm/meshery/charts/meshery-kuma/templates/NOTES.txt [hunk 2]
```diff
@@ -8,14 +8,14 @@
...
 {{- else if contains "LoadBalancer" .Values.service.type }}
      NOTE: It may take a few minutes for the LoadBalancer IP to be available.
            You can watch the status of by running 'kubectl get --namespace {{ .Release.Namespace }} svc -w {{ include "meshery-kuma.fullname" . }}'
   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ include "meshery-kuma.fullname" . }} --template "{{"{{ range (index .status.loadBalancer.ingress 0) }}{{.}}{{ end }}"}}")
-  echo http://$SERVICE_IP:{{ .Values.service.port }}
+  echo $SERVICE_IP:{{ .Values.service.port }}
 {{- else if contains "ClusterIP" .Values.service.type }}
   export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "meshery-kuma.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
-  echo "Visit http://127.0.0.1:8080 to use your application"
+  echo "Visit 127.0.0.1:8080 to use your application"
```
Comment (leecalcote) [inline install/kubernetes/helm/meshery/charts/meshery-kuma/templates/NOTES.txt:19]: Should this port number (8080) be variablized?

Comment (Aisuko) [reply install/kubernetes/helm/meshery/charts/meshery-kuma/templates/NOTES.txt:20]: we use `port-forward` command forward the pod port to localhost `8080` so we do not change the port 
```
  export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "meshery-kuma.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -o jsonpath="{.items[0].metadata.name}")
  kubectl --namespace {{ .Release.Namespace }} port-forward $POD_NAME 8080:{{ .Values.ports.http }}
  echo "Visit 127.0.0.1:8080 to use your application"
```

Comment (Aisuko) [reply install/kubernetes/helm/meshery/charts/meshery-kuma/templates/NOTES.txt:20]: And I changed the order of the commands may more sense to use

**Review by leecalcote — APPROVED** [review]
Thank you, @Aisuko

**Pull request merged**